packages/rancher-monitoring/69.8/rancher-monitoring/package.yaml

```diff
@@ -1,7 +1,7 @@
 url: https://github.com/prometheus-community/helm-charts.git
 subdirectory: charts/kube-prometheus-stack
 commit: 0c7bf42ac2265d13845ffe0c499d16e6b8cdedea
-version: 69.8.2-rancher.12
+version: 69.8.2-rancher.13
```

Comment:
There are a few PRs that need to be merged before this that makes the version go up to `rancher.15`, so this one needs to be `rancher.16`